cmd/plugin/cli/install.go

```diff
@@ -193,6 +199,11 @@ func setInstallValues(cmd *cobra.Command, config *apiv1.KubectlStorageOSConfig)
 		if err != nil {
 			return err
 		}
+		config.Spec.Install.InstallLocalPathProvisioner, err = cmd.Flags().GetBool(installer.InstallLocalPathProvisionerFlag)
```

Comment:
Out of curiosity, why does it depend on the portal manager being enabled?

Reply:
It does not, I think this is due to a dodgy rebase. I'll fix it